mypy/main.py

```diff
@@ -97,6 +97,21 @@ def type_check_only(sources: List[BuildSource], bin_dir: str, options: Options)
                        options=options)
 
 
+disallow_any_options = ['unimported']
+
+
+def disallow_any_argument_type(raw_options: str) -> List[str]:
+    flag_options = [o for o in raw_options.split(',') if len(o.strip()) > 0]
```

Comment:
That wouldn't work.  You'd need `if raw_options == ['']:`.  Also, I'd definitely still have a comment explaining when that would happen.

No, that's not what happens.  You explicitly filter out all entries in the list that have `len(o.strip()) == 0`.